custom_components/econnect_metronet/devices.py

```diff
@@ -284,3 +303,91 @@ def disarm(self, code, sectors=None):
         except CodeError as err:
             _LOGGER.error(f"Device | Credentials (alarm code) is incorrect: {err}")
             raise err
+
+    def turn_off(self, output):
+        """
+        Turn off a specified output.
+
+        Args:
+            output: The ID of the output to be turned off.
+
+        Raises:
+            HTTPError: If there is an error in the HTTP request to turn off the output.
+
+        Notes:
+            - The `element` is the sector ID used to arm/disarm the sector.
+            - This method checks for authentication requirements and control permissions
+                before attempting to turn off the output.
+            - If the output can't be manually controlled or if authentication is required but not provided,
+                appropriate error messages are logged.
+
+        Example:
+            To turn off an output with ID '1', use:
+            >>> device_instance.turn_off(1)
+        """
+        for id, item in self.items(q.OUTPUTS):
+            # Skip if it's not matching the output
+            if id != output:
+                continue
+
+            # If the output isn't manual controllable by users write an error il log
+            if item.get("control_denied_to_users"):
+                _LOGGER.error(
+                    f"Device | Error while turning off output: {item.get('name')}, Can't be manual controlled"
+                )
+                break
+
+            # If the output require authentication for control write an error il log
+            if not item.get("do_not_require_authentication"):
+                _LOGGER.error(f"Device | Error while turning off output: {item.get('name')}, Required authentication")
+                break
+
+            try:
+                element_id = item.get("element")
+                self._connection.turn_off(element_id)
+            except HTTPError as err:
+                _LOGGER.error(f"Device | Error while turning off output: {err.response.text}")
```

Comment:
Instead, this should stay an error as it means something is not working as expected.